lib/vector-config/src/schema.rs

```diff
@@ -6,18 +6,20 @@ use serde_json::{Map, Value};
 use vector_config_common::attributes::CustomAttribute;
 
 use crate::{
-    num::ConfigurableNumber, Configurable, ConfigurableString, GenerateError, Metadata, ToValue,
+    num::ConfigurableNumber, Configurable, ConfigurableRef, GenerateError, Metadata, ToValue,
 };
 
+pub fn apply_base_metadata(schema: &mut SchemaObject, metadata: Metadata) {
+    apply_metadata(&<()>::make_ref(), schema, metadata)
+}
+
 /// Applies metadata to the given schema.
 ///
 /// Metadata can include semantic information (title, description, etc), validation (min/max, allowable
 /// patterns, etc), as well as actual arbitrary key/value data.
```

Comment:
Should move this doc comment to `apply_base_metadata`.

Reply:
Fixed in f27b31f2b